lib/tasks/reset.rake

```diff
@@ -0,0 +1,30 @@
+namespace :reset do
```

Comment:
With the upgrade in bulkrax, a reset task was added. We need this corresponding bulkrax in order to run it.